make_export/create_groups_export.py

```diff
@@ -1,68 +1,75 @@
 #!/usr/bin/env python
+"""Utility to create the export for www.materialscloud.org/discover/curated-cofs.
+Since September 2020 it includes two types of groups:
+* discover_curated_cofs/05001N2 "ccs_groups" with only selected data for co2 capture and sequestration (ccs), for legacy
+* curated-cof_05001N2_v1 "full_groups" with all data
+"""
 
 import sys
-import os
+import datetime
 from aiida.orm.querybuilder import QueryBuilder
-from aiida.orm import Group, CifData
-from aiida.tools.importexport import export_zip
+from aiida.orm import Group
+from aiida.tools.importexport.dbexport import export  # Updated to AiiDA v1.3.0
 
 from figure.config import load_profile
 load_profile()
 
 TAG_KEY = "tag4"
-GROUP_DIR = "discover_curated_cofs/"
-EXPORT_NAME = "export_discovery_cof_23Jun20.aiida"
-CIFS_DIR = "./cifs_cellopt/"
+CSS_GROUP_DIR = "discover_curated_cofs/"
+EXPORT_FILE_NAME = "export_discovery_cof_{}.aiida".format(datetime.date.today().strftime(r'%d%b%y'))
 
 EXPORT = True
-PRINT_OPT_CIFS = True
-CLEAR = True
+PRINT_OPT_CIFS = False
+CLEAR = False
 
-desired_nodes = ['orig_cif', 'orig_zeopp', 'dftopt', 'opt_cif_ddec', 'opt_zeopp', 'isot_co2', 'isot_n2', 'appl_pecoal']
+css_nodes = ['orig_cif', 'orig_zeopp', 'dftopt', 'opt_cif_ddec', 'opt_zeopp', 'isot_co2', 'isot_n2', 'appl_pecoal']
 
 
-def delete_old_groups():
+def delete_css_groups():
     qb = QueryBuilder()
-    qb.append(Group, filters={'label': {'like': GROUP_DIR + "%"}})
+    qb.append(Group, filters={'label': {'like': CSS_GROUP_DIR + "%"}})
     if qb.all():
-        print("Groups '{}' found, deleting...".format(GROUP_DIR))
+        print("Groups '{}' found, deleting...".format(CSS_GROUP_DIR))
         for q in qb.all():
             group = q[0]
             group.clear()
             Group.objects.delete(group.pk)
+    else:
+        print("No previous Group {} found to delete.".format(CSS_GROUP_DIR))
 
 
-# Clear and delete old groups
-delete_old_groups()
+# Clear and delete CSS groups created in a previous session
+delete_css_groups()
 
 # Query for all the CURATED-COFs
 qb = QueryBuilder()
 qb.append(Group, filters={'label': {'like': r'curated-cof\_%\_v%'}})
-
-# Create groups for each and fill them with desired nodes
-print("Finding nodes with tag:", desired_nodes)
-
-all_new_groups = []
-for q in qb.all():
-    ref_group = q[0]
-    mat_id = ref_group.label.split("_")[1]
-    new_group = Group(label=GROUP_DIR + mat_id).store()
-    all_new_groups.append(new_group)
-    left_nodes = desired_nodes.copy()
-    for node in ref_group.nodes:
+all_full_groups = qb.all(flat=True)  #
```

Comment:
One question - did you run this?
pylint complains that `qb.all` does not allow for a `flat` keyword argument, i.e. this should fail

Reply:
Yes, then I think it was a feature introduced later. 
It is very convenient to avoid another step for getting the element from the single-element list.